test/cpp/end2end/end2end_test.cc

```diff
@@ -218,8 +226,39 @@ class TestServiceImpl : public ::grpc::testing::EchoTestService::Service {
   explicit TestServiceImpl(const grpc::string& host)
```

Comment:
hmm..this is weird..I see that you did move TestServiceImpl to a separate file in SHA: d6f7d99dc6551 but somehow, in the latest mater, it is back in end2end_test.cc

https://github.com/grpc/grpc/blob/master/test/cpp/end2end/end2end_test.cc

I guess that is why I was confused :). I am trying to see where this got added back but so far couldn't trace it (git blame on end2end_test.cc isn't helping either)..